src/index.html

```diff
@@ -23,6 +23,12 @@ <h3>Music</h3>
     <button id="music_button">Search</button>
 
     <h3>Park</h3>
+    <select id = "section_park">
+      <option value="volleyball">Volleyball</option>
+      <option value="basketballCourts">Basketball Courts</option>
+      <option value="baseballFields">Baseball Fields</option>
+      <option value="golfCourse">Golf Course</option>
+      <option value="footballFields">Football Fields</option>
     <select id="section_id">

```

Comment:
Need to delete lines 32-37. Extraneous that has been replaced by actual values above.